src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -112,18 +106,6 @@ The .NET Foundation licenses this file to you under the MIT license.
     <AutoInitializedAssemblies Include="System.Private.DisabledReflection" Condition="$(IlcDisableReflection) == 'true'" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Could you also delete the `<!-- If running from a package these values need to be set again with the resolved IlcXYZPath -->` comment in this file - we no longer set these twice. Good riddance. I never understood why it was needed.